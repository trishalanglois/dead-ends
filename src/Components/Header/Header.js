import React from 'react';
import './Header.scss';

function Header() {
  return (
    <nav className="header">
      <h1 className="main-header">DEAD ENDS</h1>
      <div className="header-line"></div>
    </nav>
  )
}

export default Header;
