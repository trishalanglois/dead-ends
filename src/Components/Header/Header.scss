@import '../../scss/variables';

.header {
  background-color: $dark-color;
  color: $light-grey;
  display: flex;
  flex-direction: column;
  align-items: center;
  // position: fixed;
}

.main-header {
  height: 2em;
  font-family: $accent-font;
  font-size: 4em;
  font-weight: 200;
  margin-bottom: -.5em;
}

.header-line {
  width: 26em;
  border-top: 1px solid $medium-grey;
  height: .5em;
}
