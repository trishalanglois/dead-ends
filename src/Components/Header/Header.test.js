import React from 'react';
import { render } from '@testing-library/react';
import Header from './Header';
import { shallow } from 'enzyme';


describe('Header', () => {
  it('should match the snapshot', () => {
    let wrapper = shallow(<Header />)

    expect(wrapper).toMatchSnapshot();
  })
})
