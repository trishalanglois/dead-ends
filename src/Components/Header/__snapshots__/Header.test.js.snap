// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot 1`] = `
<nav
  className="header"
>
  <Link
    to="/"
  >
    <h1
      className="main-header"
    >
      DEAD ENDS
    </h1>
  </Link>
  <div
    className="header-line"
  />
</nav>
`;
